aws-rds.sql

```diff
@@ -0,0 +1,150 @@
+/* Setting up the RDS database:
+
+My RDS info:
+
+RDS endpoint: (db-module11-zach-johnson.csu5hk3jzveb.us-east-2.rds.amazonaws.com)
+
+user: admin
+password: slack me for password
+
+I chose RDS for a couple of reasons, one AWS is such an important tool to have and know in the tech field these days. And 2 Amir always talks about how much he loves AWS so I had to check it out for myself. 
+
+This was not a simple task, creating the account and the instance itself was very straight forward. But when it came to connecting to the instance from workbench is where it got tricky. Had to do a lot of troubleshooting and googling to finally get it to go. https://stackoverflow.com/questions/16488135/unable-to-connect-mysql-workbench-to-rds-instance this stack thread is where I finally found the answer. To sum up, had to change the inboud security rules to allow my IP address to connect and added a third rule making it open to all. Not sure if both work since I did them both. Once I had workbench connected then it was easy to create a new schema and then start adding tables. I populated the tables with all of the data from the get go so cut out a lot of the steps from exercise 1. I could also have done a mysql dump to transfer the data over from the first exercise but wasn't sure that was what you wanted to see, so I didn't. And below is the sql commands I used: */ 
+
+create table tekcampers(
```

Comment:
in production, you'd want to restrict these more - limit nulls for the names, create linked tables for the different types of degrees or bootcamps, etc.; basically, abstract out (or prevent) repeated information